cprover_bindings/src/irep/irep.rs

```diff
@@ -47,8 +47,8 @@ impl Irep {
     /// Adds a `comment` sub to the irep.
     /// Note that there might be comments both on the irep itself and
     /// inside the location sub of the irep.
-    pub fn with_comment<T: Into<InternedString>>(self, c: T) -> Self {
-        self.with_named_sub(IrepId::Comment, Irep::just_string_id(c))
+    pub fn with_kani_comment<T: Into<InternedString>>(self, c: T) -> Self {
```

Comment:
This ends up with "kani" stuff inside cprover_bindings. Is there a better way?

Reply:
Good point.
The only other way is to use a `IrepId::FreeformString`.